improving.html

```diff
@@ -92,49 +92,49 @@
 
 	<h2 id="resources">Important Resources</h2>
 	<p>The following resources may be helpful right away to the designers and developers in your team:</p>
 	<ul>
 		<li><a href="https://www.w3.org/WAI/gettingstarted/tips/">Tips for Getting Started</a> &mdash; Practical considerations with examples for designing, writing, and developing accessible web content.</li>
-		<li><a href="https://www.w3.org/WAI/tutorials/">Web Accessibility Tutorials</a> &mdash; More comprehensive information on specific topics with guidance on addressing the accessibility requirements.</li>
+		<li><a href="https://www.w3.org/WAI/tutorials/">Web Accessibility Tutorials</a> &mdash; Detailed information on specific topics with guidance on addressing accessibility requirements.</li>
 		<li><a href="https://www.w3.org/WAI/WCAG20/quickref/">How to Meet WCAG 2.0</a> &mdash; A customizable quick reference to Web Content Accessibility Guidelines (WCAG) 2.0.</li>
 	</ul>
 
 	<h2 id="understand">What is Accessibility?</h2>
 	<p>If you are new to accessibility, it is often helpful to first get a basic idea of the topic. Here are some quick introductions:</p>
 	<ul>
-		<li><a href="https://www.w3.org/standards/webdesign/accessibility">W3C - Accessibility</a> &mdash; Introduces the essential concepts, rationale, and resources.</li>
-		<li><a href="https://www.w3.org/WAI/perspectives">Web Accessibility Perspectives</a> &mdash; Short videos that introduce accessibility features and their benefits for everyone.</li>
+		<li><a href="https://www.w3.org/standards/webdesign/accessibility">W3C - Accessibility</a> &mdash; Introduces essential concepts, rationale, and resources.</li>
+		<li><a href="https://www.w3.org/WAI/perspectives">Web Accessibility Perspectives</a> &mdash; Short non-technical videos that introduce accessibility features and their benefits for everyone.</li>
 	</ul>
 	<p>If there is a need, more in-depth introductions and background is provided in <a href="https://www.w3.org/WAI/intro/people-use-web/">How People with Disabilities Use the Web</a>. Specifically the <a href="https://www.w3.org/WAI/intro/people-use-web/principles">Accessibility Principles</a> section of the resource introduces the accessibility requirements and guidelines.</p>
 
 	<h2 id="explore">What are the Issues?</h2>
-	<p>If you already know of the specific issues you need to address, you can skip this section. If you are starting to address accessibility from new, the following manual helps you get a rough idea of the current situation on your website:</p>
+	<p>If you already know the specific issues you need to address, you can skip this section. If you are starting to address accessibility without this information, the following manual helps you get a rough idea of the current situation on your website:</p>
```

Comment:
Yeah I went back and forth on that, I originally had "from scratch" but wasn't sure that idiom was used internationally.  I also considered "from the ground up" but felt that sounded like they haven't even built the site, so I think "without background" works. :)